src/makeRobotAccountant.js

```diff
@@ -7,6 +7,25 @@
 
 function makeRobotAccountant() {
   // write code here
+  let leftCalls = 0;
+
+  const calc = (x) => {
+    const sum = (y) => {
+      if (leftCalls >= 3) {
+        leftCalls--;
```

Comment:
it's not clear why do you decrease this variable
will be better to specify needed check (for odd and even numbers) inside the if statement - for each moment check is current value is odd or even, and go through needed flow